src/Plugin/OgFields/AudienceField.php

```diff
@@ -34,6 +34,10 @@ public function getFieldStorageConfigBaseDefinition(array $values = array()) {
       'settings' => [
         'target_type' => $this->getEntityType(),
       ],
+      // @todo Is this correct? The description of OgStandardReferenceItem says:
```

Comment:
Ok, removed the comment, the updating of the description can be handled in #177.